components/brave_rewards/browser/rewards_service_browsertest.cc

```diff
@@ -308,6 +330,49 @@ class BraveRewardsBrowserTest
       verified.c_str());
   }
 
+  static std::vector<double> GetSiteBannerTipOptions(
+      content::WebContents* site_banner) {
+    auto options = content::EvalJs(
+        site_banner,
+        R"(
+            const delay = t => new Promise(resolve => setTimeout(resolve, t));
+            delay(500).then(() => Array.prototype.map.call(
+                document.querySelectorAll(
+                    "[data-test-id=amount-wrapper] div span"),
+                node => parseFloat(node.innerText)))
+        )",
+        content::EXECUTE_SCRIPT_DEFAULT_OPTIONS,
+        content::ISOLATED_WORLD_ID_CONTENT_END).ExtractList();
+
+    std::vector<double> result;
+    for (const auto& value : options.GetList()) {
+      result.push_back(value.GetDouble());
+    }
+    return result;
+  }
+
+  static std::vector<double> GetRewardsPopupTipOptions(
+      content::WebContents* popup) {
+    auto options = content::EvalJs(
+        popup,
+        R"_(
```

Comment:
why do we do `_( .... )_`?

Reply:
prefix(optional) R "delimiter( raw_characters )delimiter"	(6)	(since C++11)

Reply:
The additional `_`s were needed because the selector contains `)"` which is the string termination sequence.